### DISCUSSION TITLE
Advice needed on 'HandlebarsConfiguration.UnresolvedBindingFormatter' is obsolete

### DISCUSSION THREAD

**Discussion by StefH** [Handlebars-Net/Handlebars.Net#425] [Q&A] [answered]
I get this message
```
'HandlebarsConfiguration.UnresolvedBindingFormatter' is obsolete: 'Register custom formatters using `Formatters` property'
```

With this c# code:
``` c#
private static readonly HandlebarsConfiguration HandlebarsConfiguration = new HandlebarsConfiguration
        {
            UnresolvedBindingFormatter = "{0}"
        };
```

**Comment by oformaniuk** [comment, marked as answer]
Hello @StefH 
See [Register custom value formatter](https://github.com/Handlebars-Net/Handlebars.Net#register-custom-value-formatter) section of the readme.
Default implementation for UndefinedBindingResult you can find in [UndefinedFormatter.cs](https://github.com/Handlebars-Net/Handlebars.Net/blob/master/source/Handlebars/IO/Formatters/UndefinedFormatter.cs)

**Comment by StefH** [comment]
Thanks
